zarnitsa/DataAugmenterInternally.py

```diff
@@ -55,13 +63,34 @@ def augment_column(
         to_aug = augment_column_method[aug_type](col, **kwargs)
         return to_aug if return_only_aug else pd.concat([not_to_aug, to_aug])
 
+    def _apply_imputation_knn(self, data):
```

Comment:
why it is imputation_knn if we initialize here both cases - simple and knn?

Reply:
my bad, fixed